setup.cfg

```diff
@@ -28,13 +28,13 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     Babel>=2.8
-    Flask-BabelEx>=0.9.4
-    invenio-base>=1.2.11
+    Flask-Babel>=2.0.0
+    invenio-assets>=1.3.0<1.4.0
+    Flask-Login>=0.6.2
```

Comment:
I tried to remove all three. It is not working. They have to be required otherwise they would not be installed. But we could add it to the tests_require? I would only move `invenio-assets`and `flask-login` to `tests_require`. `Babel` and `Flask-Babel`are the key dependencies of this package in my opinion and should therefore remain in `install_requires

Reply:
In the `install_requires`, we should add any dependency that is imported in the code, any import.
If extra dependencies are imported in tests, then they should go in the tests_require.

Reply:
You are probably right...